packages/presets/graphql-modules/src/utils.ts

```diff
@@ -139,14 +139,19 @@ export function groupSourcesByModule(sources: Source[], basePath: string): Recor
   const grouped: Record<string, Source[]> = {};
 
   sources.forEach(source => {
-    // PERF: we could guess the module by matching source.location with a list of already resolved paths
-    const mod = extractModuleDirectory(source.name, basePath);
+    const normalizedFilepath = normalize(source.name);
```

Comment:
this is a bit duplicated?

Maybe I'm wrong but is there a chance in `extractModuleDirectory` that would save this duplication and we could just call that?

Reply:
You're right. I thought of moving it to a separate function but ignored it later since it did not make sense for me to make a separate function for 3 lines of code. But maybe it makes sense over time. Will update this PR.